### PR TITLE
Block and non block page titles

### PR DESCRIPTION
**Pull request by JamesGardiner**
### What is the context of this PR?
Adds `page_title` key to the list of extractable strings

### CODE REVIEW

File: tests/schemas/ga/test_language.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
         {
             "id": "default-section",
             "title": "Sonra\u00ed t\u00ed",
+            "page_title": "Custom section page title",
```
Comment (MebinAbraham) [inline tests/schemas/ga/test_language.json:33]: Examples should be translated.

Comment (JamesGardiner) [reply tests/schemas/ga/test_language.json:33]: As in formally or as in via @berroar ?

Comment (MebinAbraham) [reply tests/schemas/ga/test_language.json:33]: Google is fine.

File: tests/schemas/cy/test_language.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
         {
```
Comment (MebinAbraham) [inline tests/schemas/cy/test_language.json:30]: I guess this wasn't translated using the translation process as it is missing keys page title keys.

Comment (JamesGardiner) [reply tests/schemas/cy/test_language.json:30]: Addressed in e3b8c52 for `cy` and `ga` schemas

File: eq_translations/extractable_strings.py [hunk 3]
```diff
@@ -7,6 +7,7 @@
     {"json_path": "$.submission.title", "description": "Submission title"},
     {"json_path": "$.submission.warning", "description": "Submission warning"},
     {"json_path": "$.sections[*].title", "description": "Section title"},
+    {"json_path": "$..page_title", "description": "Custom page title"},
```
Comment (ajmaddaford) [inline eq_translations/extractable_strings.py:10]: This should probably just be `Page title` as I don't think the fact that it is custom will be relevant to a translator.

**Review by ajmaddaford — APPROVED** [review]

**Review by MebinAbraham — COMMENTED** [review]
The po files don't match the new extraction. I.e it still says "Custom ..." etc,

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**